<launch>

	<node
		pkg = "turtlesim"
		type = "turtlesim_node"
		name = "turtlesim"
		respawn = "true"
	/>

	<node
		pkg = "turtlesim"
		type = "turtle_teleop_key"
		name = "teleop_key"
		required = "true"
		launch-prefix = "xterm -e"
	/>
	
	<node
		pkg = "packagename"
		type = "publish_velocity_executable"
		name = "publish_velocity"
	>
		<param name="max_vel" value="3" />
	</node>

	<node
		pkg ="packagename"
		type = "set_background_color_executable"
		name = "set_background_color"
	/>

</launch>
